includes/notes/class-wc-admin-notes-order-milestones.php

```diff
@@ -230,6 +165,103 @@ public function get_current_milestone() {
 		return $milestone_reached;
 	}
 
+	/**
+	 * Get the appropriate note title for a given milestone.
+	 *
+	 * @param int $milestone Order milestone.
+	 * @return string Note title for the milestone.
+	 */
+	public function get_note_title_for_milestone( $milestone ) {
+		switch ( $milestone ) {
+			case 1:
+				return __( 'First order', 'woocommerce-admin' );
+			case 10:
+			case 100:
+			case 250:
+			case 500:
+			case 1000:
+			case 5000:
+			case 10000:
+			case 500000:
+			case 1000000:
+				return sprintf(
+					/* translators: Number of orders processed. */
+					__( 'Congratulations on processing %s orders!', 'woocommerce-admin' ),
+					wc_format_decimal( $milestone )
+				);
+			default:
+				return '';
+		}
+	}
+
+	/**
+	 * Get the appropriate note content for a given milestone.
+	 *
+	 * @param int $milestone Order milestone.
+	 * @return string Note content for the milestone.
+	 */
+	public function get_note_content_for_milestone( $milestone ) {
+		switch ( $milestone ) {
+			case 1:
+				return __( 'Congratulations on getting your first order from a customer! Learn how to manage your orders.', 'woocommerce-admin' );
```

Comment:
The last part of this note (Learn how to manage your orders) seems out of place? Any reason why this text cannot be the CTA text in `get_note_action_for_milestone`?

Reply:
I _think_ the exact verbiage here was outlined by Pi/Marketing - but seems like a valid change.